docs/resources/azure_monitor_log_profile.md.erb

```diff
@@ -0,0 +1,88 @@
+---
+title: About the azure_monitor_log_profile Resource
+platform: azure
+---
+
+# azure\_monitor\_log\_profile
+
+Use the `azure_monitor_log_profile` InSpec audit resource to test properties
+of an Azure Monitor Log Profile.
+
+<br />
+
+## Syntax
+
+An `azure_monitor_log_profile` resource block identifies a Log Profile by name.
+
+    describe azure_monitor_log_profile(name: 'default') do
+      ...
+    end
+<br />
+
+## Examples
+
+### Test that a Log Profile exists
+
+    describe azure_monitor_log_profile(name: 'default') do
+      it { should exist }
+    end
+
+### Test that a Log Profile retention is enabled
+
+    describe azure_monitor_log_profile(name: 'default') do
+      its('retention_enabled') { should be true }
+    end
+
+<br />
+
+## Parameters
+
+* `name`
+
+## Parameter Examples
+
+The name of the Log Profile.
+
+    describe azure_monitor_log_profile(name: 'default') do
+      it { should exist }
+    end
+
+## Properties
+
+* `retention_enabled`
+* `retention_days`
+
+### retention_enabled
+
+Determine if retention is enabled
+
+    its('retention_enabled') { should be true }
+
+### retention_days
```

Comment:
`s/retention_days/retention\_days` - I've been escaping all underscores just to be safe. If down the road another underscore was added to this line it would add emphasis to everything between them.